docs/api/session.md

```diff
@@ -91,6 +91,15 @@ session.defaultSession.on('will-download', (event, item, webContents) => {
 })
 ```
 
+#### Event: 'preconnect'
+
+Returns:
+
+* `event` Event
+* `preconnectUrl` string
+
+Emitted when Electron is about to preconnect `preconnectUrl`.
```

Comment:
This isn't quite accurate: if the JS doesn't do anything, Electron won't preconnect to anything. Perhaps:

> Emitted when a render process requests preconnection to a URL.